### PR TITLE
Change

### PR DESCRIPTION
**Pull request by Ebubeker**

### CODE REVIEW

File: app/client/src/components/isochrones/Isochrones.vue [hunk 1]
```diff
@@ -660,9 +660,11 @@
...
                               >fas fa-clock
                             </v-icon>
-                            <span class="caption">{{
-                              calculation.config.settings.weekday
-                            }}</span>
+                            <span class="caption"
+                              >{{ publicTransport.fromTime }}
+                              -
+                              {{ publicTransport.toTime }}</span
+                            >
```
Comment (majkshkurti) [inline app/client/src/components/isochrones/Isochrones.vue:667]: This should come from the calculation object as it should not change when the user selects a new time range. 
The method `secondsToHoursAndMins` can directly be called here 
e.g. `secondsToHoursAndMins(calculation.config.settings.from_time)`
To use the function in the template, you should add it to the methods object like below: 

```
 methods: {
    secondsToHoursAndMins, <-----
    renderRadarChart: function() {
      let labels = this.selectedPoisOnlyKeys;
      let labels;
      if (this.chartDatasetType === 1) {
     ....................

```

Comment (Ebubeker) [reply app/client/src/components/isochrones/Isochrones.vue:667]: ok, I will do the changes

File: app/client/src/components/isochrones/Isochrones.vue [hunk 2]
```diff
@@ -1528,6 +1533,12 @@ export default {
                 surfaceData: singleValuedSurface,
                 feature: olFeatures[0]
               };
+              let calcTimeFrom = calculation.config.settings.from_time;
+              let calcTimeTo = calculation.config.settings.to_time;
+              this.publicTransport.fromTime = secondsToHoursAndMins(
+                calcTimeFrom
+              );
+              this.publicTransport.toTime = secondsToHoursAndMins(calcTimeTo);
```
Comment (majkshkurti) [inline app/client/src/components/isochrones/Isochrones.vue:1541]: We can remove this part completely

**Review by majkshkurti — COMMENTED** [review]
Thanks.

**Review by majkshkurti — APPROVED** [review]

**Pull request merged**